docs/src/main/paradox/cluster-sharding.md

```diff
@@ -3,11 +3,11 @@ project.description: Alpakka Kafka provides a module to use Kafka with Akka Clus
 ---
 # Akka Cluster Sharding
 
-Akka Cluster allows the user to use an @extref[external shard allocation](akka26:/typed/cluster-sharding.html#external-shard-allocation) strategy in order to give the user more control over how many shards are created and what cluster nodes they are assigned to. 
+Akka Cluster allows the user to use an @extref[external shard allocation](akka:/typed/cluster-sharding.html#external-shard-allocation) strategy in order to give the user more control over how many shards are created and what cluster nodes they are assigned to. 
 If you consume Kafka messages into your Akka Cluster application then it's possible to run an Alpakka Kafka Consumer on each cluster node and co-locate Kafka partitions with Akka Cluster shards. 
 When partitions and shards are co-located together then there is less chance that a message must be transmitted over the network by the Akka Cluster Shard Coordinator to a destination user sharded entity.
 
-This module directly depends on `akka-cluster-sharding-typed` version 2.6.6 or later.
+This module directly depends on `akka-cluster-sharding-typed`.
```

Comment:
We should list it explicitly in the `@@dependency` with a symbolic Akka version.